packages/mui-base/src/Slider/Indicator/useSliderIndicator.ts

```diff
@@ -21,9 +19,19 @@ const axisProps = {
   },
 };
 
-function useSliderIndicator(
-  parameters: UseSliderIndicatorParameters,
-): UseSliderIndicatorReturnValue {
+/**
```

Comment:
Hooks are not documented, so this header is invalid. The current docs:api script doesn't create it anymore.